src/main/java/io/mosip/mimoto/exception/InvalidIssuerIdException.java

```diff
@@ -0,0 +1,24 @@
+package io.mosip.mimoto.exception;
+
+public class InvalidIssuerIdException extends BaseUncheckedException {
+
+    /**
+     * Unique id for serialization
+     */
+    private static final long serialVersionUID = -5350213197226295789L;
```

Comment:
serialVersionUID's use case yet to be figured out.
(This UID in code is copied from InvalidInputException)

Reply:
Updated serialVersionUID